numberGame.js

```diff
@@ -0,0 +1,16 @@
+export const errorMessageNoGuessesLeft = 'No guesses left';
+export class Guesser {
+    constructor(number, lives) {
+        this.number = number;
+        this.lives = lives;
+    }
+
+    guess(number) {
+        if (this.lives <= 0) throw new Error(errorMessageNoGuessesLeft);
```

Comment:
similar logic should live in a constructor to not allow creating instance with zero/negative lives

Reply:
update: should is the wrong word - COULD

Reply:
I've decided not to do that